Makefile

```diff
@@ -209,18 +209,19 @@ go-build: pilot istioctl pilot-agent sidecar-initializer mixs mixc node-agent is
 # Target: go test
 #-----------------------------------------------------------------------------
 
-.PHONY: go-test localTestEnv testApps
+.PHONY: go-test localTestEnv test-bins
 
 GOTEST_PARALLEL ?= '-test.parallel=4'
 GOTEST_P ?= -p 1
 GOSTATIC = -ldflags '-extldflags "-static"'
 
-testApps:
-	CGO_ENABLED=0 go build ${GOSTATIC} -o ${GOPATH}/bin/server istio.io/istio/pilot/test/server
-	CGO_ENABLED=0 go build ${GOSTATIC} -o ${GOPATH}/bin/client istio.io/istio/pilot/test/client
-	CGO_ENABLED=0 go build ${GOSTATIC} -o ${GOPATH}/bin/eurekamirror istio.io/istio/pilot/test/eurekamirror
+test-bins:
+	CGO_ENABLED=0 go build ${GOSTATIC} -o ${GOPATH}/bin/pilot-test-server istio.io/istio/pilot/test/server
+	CGO_ENABLED=0 go build ${GOSTATIC} -o ${GOPATH}/bin/pilot-test-client istio.io/istio/pilot/test/client
```

Comment:
can you call it something shorter? it's pain to type to debug pod containers

Reply:
Its still called client and server.. the locally stored file in host is renamed for clarity. See Dockerfile..

Reply:
that's the one that matters. the workflow is ssh-ing into pod and calling it out by typing the binary name.

Reply:
ok, so it's renamed back to client in docker. i don't understand the rename then, since the binary is never invoked outside of docker.

Reply:
Its so that we don't pollute a user's machine when they do make build/make pilot/etc..